packages/devops/scripts/ci/validateTests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+DIR=`dirname "$0"`
+ROOT="${DIR}/../../../../"
+
+node ${ROOT}/scripts/node/validateTests
```

Comment:
So the validation scripts are a bit scattered around now... This one, defined in `devops` invokes another one defined in the root.

It will make things simpler to move all those in one location, probably `validate...` ones (which are not only devops specific) under the root? Or we could create a `scripts` package, which can also include reusable scripts (eg `build` for internal packages). This would be similar to how `react-scripts` is used by `create-react-app` projects.